lua-docs/content/guides/tutorials/tutoworld.yml

```diff
@@ -0,0 +1,1036 @@
+        end
+
+        Client.Tick = function(dt)
+            -- Game loop, executed ~30 times per second on each client.
+
+            -- Detect if player is falling,
+            -- drop it above the map when it happens.
+            if Player.Position.Y < -500 then
+                dropPlayer()
+                Player:TextBubble("💀 Oops!")
+            end
+        end
+
+        -- jump function, triggered with Action1
+        Client.Action1 = function()
+            if Player.IsOnGround then
+                Player.Velocity.Y = 100
+            end
+        end
+
+        --
+        -- Server code
+        --
+
+        Server.Tick = function(dt) 
+            -- Server game loop, executed ~30 times per second on the server.
+        end
+
+    - text: |
+        The code is quite self-explanatory for the most part, and you can also check out the [Beginner's Guide](https://particubes.com/beginners-guide) for a quick peek at the overall structure... but don't worry: throughout the rest of this tutorial, we'll modify or expand on this snippet and gradually dive into its different parts.
+
+    - title: "Setting the world's map"
+
+    - text: |
+        First of all, let's look at the very first lines of the script:
+
+    - code: |
+        Config = {
+            Map = "aduermael.hills"
+        }
+
+    - text: |
+        This [Config](/reference/config) object is where we set the global parameters of our world, and most importantly the map to use and the items to import. By default, the auto-generated code loads one of the maps made by the team, which is referenced by the code: `aduermael.hills`.
+
+        This is a good opportunity to say two important things about map and items in Particubes:
+
+        1. they are all shared to every user in Particubes - this means that, as soon as you know the reference of an asset, you can import and use it in your own game directly
+        2. all map or item references in Particubes are built with the same format: `creator_name.item_name`. So, here, `aduermael` is the creator, and `hills` is the name of the map to import.
+
+        If you want to get started quickly, you can use some of the maps already prepared by our team:
+
+        - `aduermael.hills`
+        - `aduermael.mountains_cabin`
+        - `aduermael.lovely_castle`
+        - `aduermael.red_planet`
+        - `aduermael.rockies`
+        - `claire.desert`
+        - `claire.market`
+
+        But, of course, you can also create your own! To do so, just go to the Item editor and assemble voxels to your liking as explained in the [Beginner's Guide](https://particubes.com/beginners-guide) :)
+
+        <i>Note: at this time, there isn't a dedicated map editor yet. So you simply make a map "item", and then import it as the map of your world using the [`Config`](/reference/config) object.</i>
+
+        For example, in my case, I prepared a small map called `minadune.tutoworld_map`:
+
+        // gif tutoworld map turnaround
```

Comment:
do you want to add a GIF here?

Reply:
Oups, nope - I forgot to add my .mp4 file, actually! ^^ (first thought I'd use GIFs but the engine was already made for movies and it's easier to prepare :)

Thanks, nice catch - I'll fix it!

Reply:
great ! I remember now, we chose video instead of GIF for the compression (smaller files)